pipeline/train/configs/model/student.base.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/browsermt/students/tree/master/train-student/models/student.base
```

Comment:
Suggestion: Either duplicate some of the notes about the "Ludicrously Fast" paper here, or link to the `.md` docs, e.g. `"For more details see: [link]"`